data-reconciliation-app/test/README.md

```diff
@@ -0,0 +1,76 @@
+
+![Testing Result](../docs/images/test-result.jpg)
+
+### How to run unit tests
+
+To run the application unit tests
+
+```bash
+# In the terminal window
+cd data-reconciliation-app     # Navigate to reconciliation sample folder
+make unit-test                 # Build and run unit-test using jest framework
+```
+
+**NOTE**: To enable the CCF functionality on testing environment, the application is using built-in feature of CCF, [the Polyfill implementation](https://microsoft.github.io/CCF/main/js/ccf-app/modules/polyfill.html) which is override the CCF modules implementation to support the testing and local environments
+
+## End to end testing
+
+End to end testing (E2E testing) is a testing method that involves testing an application’s workflow from beginning to end. This method aims to replicate real user scenarios to validate the system for integration and data integrity.
+
+The application uses bash scripts and curl to perform end-to-end testing for the application endpoints.
+
+There are 3 different types of network this sample can be tested against:
+- Sandbox (virtual)
+- Docker (virtual - enclave)
+- Azure Managed CCF (enclave)
+
+
+The test script [./test.sh](./test.sh) contains the application test scenarios and result validation for each endpoint. All requests are made using curl and assertion logic in a bash script to validate the results and generate a testing report.
+
+
+**Note:** _The test script of each sample is called by wrapper scripts that exist in the root [scripts](../../scripts/) folder. 
+These wrapper scripts are used for all samples, and they are responsible for starting the particular network with the correct constitution and setting up governance (users, members, and applications). The wrapper scripts will also close the network after the tests have finished (excluding mCCF)._
+
+
+### How to run e2e-test
+
+To run the application e2e tests
+
+```bash
+# In the terminal window
+cd data-reconciliation-app     # Navigate to reconciliation sample folder
+make test                      # Run end-to-end(e2e) tests in a sandbox (virtual) environment
+```
+
+|   Network   |     Command                |                  Script                       | Supported environment     |
+| :---------  | :------------------------- | :-------------------------------------------  | :------------------------ |
+| Sandbox     | `make test`                | [test_sandbox](../../scripts/test_sandbox.sh) | virtual                   |
+| Docker      | `make test-docker-virtual` | [test_docker](../../scripts/test_docker.sh)   | virtual                   |
+| Docker      | `make test-docker-enclave` | [test_docker](../../scripts/test_docker.sh)   | enclave (TEE)             |
+| Managed CCF | `make test-mccf`           | [test_mccf](../../scripts/test_docker.sh)     | enclave (TEE)             |
```

Comment:
when looking this file in preview mode, the links for the script files seem broken.
They seem to be redirecting to your own branch: `https://github.com/microsoft/ccf-app-samples/blob/aym/testing-strategy/scripts/test_sandbox.sh`
Not sure if this is a general issue or just on my end.

Please double check.

Reply:
it is working correctly with me, it's may be preview issue